src/main/kotlin/no/nav/syfo/behandler/domain/Behandler.kt

```diff
@@ -38,3 +38,5 @@ fun Behandler.toBehandlerDialogmeldingDTO() = BehandlerDialogmeldingDTO(
     poststed = this.poststed,
     telefon = this.telefon,
 )
+
+fun Behandler.hasRequiredIds(): Boolean = personident != null || herId != null || hprId != null
```

Comment:
Jeg synes det høres ut som at behandleren må ha flere id'er her 🤷‍♂️ Egentlig mener man vel "`behandler har minst én id`"?

Reply:
Sant. Har vi noe begrep for disse id-ene? Føler hasAtLeastOneId blir noe vagt🤔

Reply:
Hmm, det vet jeg ikke 🤔 Enig at den første er vag, men å ramse opp alle typene i metodenavnet er jo også litt kjedelig 🤷‍♂️

Reply:
Hva med hasId, hasAnId eller anIdExists eller noe i den duren?